test/unix/force_merge.ml

```diff
@@ -169,11 +169,35 @@ let readonly_and_merge () =
   loop 10;
   test_fd ()
 
+let test_iter () =
+  let context_with_duplicates () =
+    let { Context.rw; _ } = Context.full_index ~size:4 ~log_size:4 () in
+    let k = Key.v () in
+    let v1 = Value.v () in
+    Index.replace rw k v1;
```

Comment:
Not strictly a test of `force_merge`, but I think it's OK for it to be in here. Can you add a comment here to say that this `replace` induces a merge operation (and the subsequent one does not)? The interaction between `size` and `log_size` may not be so obvious to a future reader :slightly_smiling_face: